R/colour-text.r

```diff
@@ -1,8 +1,9 @@
-
+# fix the check NOTE:Namespace in Imports field not imported from: ‘crayon’
```

Comment:
It's not really necessary (I'd argue that the warning is spurious because it doesn't capture build time dependencies) but getting rid of a NOTE is usually worth it.